### DISCUSSION TITLE
Einige Dropdownfelder im Admincenter enden mit 2020

### DISCUSSION THREAD

**Discussion by niclasd** [scheb/open-apexx#62] [General]
Hallo zusammen,

zunächst einmal Danke für den jahrelangen Support - auch wenn ich das System selbst nicht mehr nutze, hatte ich apexx auf meiner eigenen Seite lange im Einsatz und es war in vielerlei Hinsicht den damaligen Alternativen überlegen. Die Entscheidung, "open apexx" nun einzustellen kann ich nachvollziehen, aber trotzdem "hoffe" ich noch auf eine kleine Rückmeldung, da ein guter Freund das System weiterhin im Einsatz hat und von mir technisch unterstützt wird :) 

Wir haben festgestellt, dass manche Dropdownfelder "nur" bis ins Jahr 2020 gehen (Wer hätte das auch damals bei der Entwicklung ahnen können) ;-)

Screenshot: https://abload.de/img/unbenannt-16bj19.jpg

Erste Idee: Einfach weitere Jahre analog der Vorjahre in den Code eintragen, allerdings finde ich ich über die Code-Suche lediglich Archiv-Templates: https://github.com/scheb/open-apexx/search?q=2019

Hat jemand eine Idee?

**Comment by bigfraggle** [comment]
Gute Frage, muss ich auch mal schauen...

**Comment by bigfraggle** [comment]
Also in modules/news/admin wird in den jeweiligen Templates auf {STARTTIME}, {ENDTIME} und {STICKYTIME} zugegriffen.
Wo diese wiederum ihre Inhalte herholen, habe ich noch nicht herausgefunden...

**Comment by DOHRMANN-SOFTWARE** [comment]
@niclasd 
Öffne die Datei 'lib/functions.admin.php' ( https://github.com/scheb/open-apexx/blob/master/src/lib/functions.admin.php#L195 ) in deinem Apexx-Verzeichnis und gehe in die Zeile 195 und ändere dort dir '2020' durch einen gewünschten Wert von dir.

Oder du kommentierst die Zeilen 194, 195 und 196 aus und fügst darunter
`$string.='<input class="input" name="t_year_'.$id.'" type="text" value="'.$date['year'].'" maxlength="4" style="width:50px"> - ';`
ein.

**Comment by bigfraggle** [comment]
Danke!

**Comment by niclasd** [comment]
Super, vielen Dank an euch beide! :)
